src/clientdlg.cpp

```diff
@@ -1275,11 +1220,7 @@ void CClientDlg::Disconnect()
     TimerDetectFeedback.stop();
     bDetectFeedback = false;
 
-    //### TODO: BEGIN ###//
-    // is this still required???
-    // immediately update status bar
-    OnTimerStatus();
-    //### TODO: END ###//
+    UpdateDisplay();
```

Comment:
TODO was removed and replaced by OnTimerStatus()
clientdlg.h shows that OnTimerStatus() just calls UpdateDisplay().